RecoTracker/IterativeTracking/python/LowPtTripletStep_cff.py

```diff
@@ -274,12 +277,17 @@
 lowPtTripletStep.mva.GBRForestLabel = 'MVASelectorIter1_13TeV'
 lowPtTripletStep.qualityCuts = [-0.6,-0.3,-0.1]
 
-#LWTNN selector
+trackingPhase1.toReplaceWith(lowPtTripletStep, TrackMVAClassifierPrompt.clone(
+     mva = dict(GBRForestLabel = 'MVASelectorLowPtTripletStep_Phase1'),
+     src = 'lowPtTripletStepTracks',
+     qualityCuts = [-0.4,0.0,0.3]
+))
+
 from RecoTracker.FinalTrackSelectors.TrackLwtnnClassifier_cfi import *
 from RecoTracker.FinalTrackSelectors.trackSelectionLwtnn_cfi import *
-trackingPhase1.toReplaceWith(lowPtTripletStep, TrackLwtnnClassifier.clone(
-     src = 'lowPtTripletStepTracks',
-     qualityCuts = [0.2, 0.5, 0.8],
+trackdnn.toReplaceWith(lowPtTripletStep, TrackLwtnnClassifier.clone(
+    src = 'lowPtTripletStepTracks',
+    qualityCuts = [0.2, 0.425, 0.75]
 ))
 
 fastSim.toModify(lowPtTripletStep, vertices = "firstStepPrimaryVerticesBeforeMixing")
```

Comment:
it may be useful to move this after pp_on_AA

Reply:
Moved to after pp_on_AA